conf/capsule.yaml.template

```
@@ -10,6 +10,8 @@ CAPSULE:
     # The base os rhel version where the capsule installed
     # RHEL_VERSION:
   # The Ansible Tower workflow used to deploy a capsule
-  DEPLOY_WORKFLOW: deploy-capsule
+  DEPLOY_WORKFLOW:
```

Now that this is a parent of multiple workflows, I think this reads better.
This suggestion also applies to the server config
```suggestion
  DEPLOY_WORKFLOWS:
```

---

Thanks for the suggestion. Applied.